HW11/homework11.py

```diff
@@ -0,0 +1,122 @@
+# 1. Додати обробку помилок до коду з завдання №7 про телефонну книгу
+# (тема: “Колекції та структури даних. Part 1”)
+# Повинно бути як мінімум два блоки try except, де їх використовувати — вирішуєте самі.
+# 2. (необов'язкове виконання) Написати кастомний Exception клас, MyCustomException,
+# який має повідомляти "Custom exception is occurred".
+
+class MyCustomException(Exception):
+    def __init__(self, message="Custom exception is occurred"):
+        self.message = message
```

Comment:
nit: this line can be deleted because the message is set in `Exception` constructor